Packs/SilentPush/Integrations/SilentPush/SilentPush.yml

```diff
@@ -1,109 +1,1936 @@
+category: Data Enrichment & Threat Intelligence
+description: The Silent Push Platform uses first-party data and a proprietary scanning engine to enrich global DNS data with risk and reputation scoring, giving security teams the ability to join the dots across the entire IPv4 and IPv6 range, and identify adversary infrastructure before an attack is launched. The content pack integrates with the Silent Push system to gain insights into domain/IP information, reputations, enrichment, and infratag-related details. It also provides functionality to live-scan URLs and take screenshots of them. Additionally, it allows fetching future attack feeds from the Silent Push system.
 commonfields:
   id: SilentPush
   version: -1
-  name: SilentPush
+name: SilentPush
+display: SilentPush
+configuration:
+- display: Base URL
+  name: url
+  type: 0
+  required: true
+  defaultvalue: https://api.silentpush.com
+- display: API Key
+  name: credentials
```

Comment:
If you are marking this as credentials then you should also consider to hide the username in case he wants to switch to Credentials and not enter the API here.

Reply:
hiddenusername: false